client-src/elements/chromedash-feature-detail.js

```diff
@@ -582,27 +587,44 @@ class ChromedashFeatureDetail extends LitElement {
     }
     const isActive = this.feature.active_stage_id === feStage.id;
 
-    // Show a button to add a trial extension stage for origin trial stages.
+    // Show any buttons that should be displayed at the top of the detail card.
     let addExtensionButton = nothing;
+    let editButton = nothing;
+    let visitTrialButton = nothing;
     if (this.canEdit && STAGE_TYPES_ORIGIN_TRIAL.has(feStage.stage_type)) {
       // Button text changes based on whether or not an extension stage already exists.
       const extensionAlreadyExists = (feStage.extensions && feStage.extensions.length > 0);
       const extensionButtonText = extensionAlreadyExists ?
         'Add another trial extension' : 'Add a trial extension';
       addExtensionButton = html`
-      <sl-button size="small" style="float:right"
+      <sl-button size="small"
           @click=${() => this.createExtensionStage(feStage, extensionAlreadyExists)}
           >${extensionButtonText}</sl-button>`;
     }
-
-    const editButton = html`
-      <sl-button size="small" style="float:right"
-          href="/guide/stage/${this.feature.id}/${processStage.outgoing_stage}/${feStage.id}"
-          >Edit fields</sl-button>
-`;
+    if (this.canEdit) {
+      editButton = html`
+        <sl-button size="small"
+            href="/guide/stage/${this.feature.id}/${processStage.outgoing_stage}/${feStage.id}"
+            >Edit fields</sl-button>`;
+    }
+    // If we have an origin trial ID associated with the stage, add a link to the trial.
+    if (feStage.origin_trial_id) {
+      let originTrialsURL = `https://origintrials-staging.corp.google.com/origintrials/#/view_trials/${feStage.origin_trial_id}`;
+      // If this is the production host, link to the production OT site.
+      if (window.location.host === 'chromestatus.com') {
+        originTrialsURL = `https://developer.chrome.com/origintrials/#/view_trials/${feStage.origin_trial_id}`;
```

Comment:
@jrobbins I wasn't sure of the best way to handle this differentiation of the staging and production OT URLs - maybe you have some opinions on this? Also, non-Googlers will not have access to the OT staging URL when developing, since it is a corp site. Hopefully this is still fine to reference in test environments?

Reply:
I think checking window.location.host falls short because we want to be able to test on non-live GAE versions in the prod GAE project, which would have a host of HASH-dot-cr-status.appspot.com.  So, another way to do it would be to check the appTitle property that is passed down as far as  chromedash-feature-page but could be passed open more level to chromedash-feature-detail.

I think that it is OK to link from cr-status-staging to the OT staging site, even if non-googlers cannot follow the links.